integrationTests/chainSimulator/bridge/esdtSafe_test.go

```diff
@@ -679,14 +686,17 @@ func deployReceiverContractInAllShards(t *testing.T, cs chainSim.ChainSimulator)
 }
 
 func isNft(esdtType core.ESDTType) bool {
-	return esdtType == core.NonFungible ||
-		esdtType == core.NonFungibleV2 ||
-		esdtType == core.DynamicNFT
+	return esdtType == 1 || // NFT
```

Comment:
Everywhere in the code, please add an enum with const values for these types and use them instead of numbers with comments